src/pages/SetupPlot.vue

```diff
@@ -262,9 +261,10 @@ export default defineComponent({
       })
       if (this.defaultPath != this.plotDirectory)
         await native.createDir(this.plotDirectory)
-      const public_key = await startFarming(this.plotDirectory)
+      const farmer_identity = await startFarming(this.plotDirectory)
       // TODO: find a way to store and retrieve the public key from client.ts.
-      LocalStorage.set("farmerPublicKey", public_key)
+      LocalStorage.set("farmerPublicKey", farmer_identity.public_key)
+      LocalStorage.set("mnemonic", farmer_identity.mnemonic)
```

Comment:
Storing mnemonic here this way is not a great idea, but can be dealt with later as things will change anyway.